Sources/Orbit/Foundation/Elevations/Elevation.swift

```diff
@@ -0,0 +1,97 @@
+import SwiftUI
+
+/// Use elevation to bring content closer to users.
+///
+/// Elevation levels with higher numbers are usually visually closer to the user.
+public enum Elevation {
+    case level1
+    case level2
+    case level3
+    case level4
+}
+
+public extension View {
+
+    /// Elevates the view to make it more prominent.
+    func elevation(_ level: Elevation, cornerRadius: CGFloat = 0) -> some View {
+        modifier(ElevationModifier(level: level, cornerRadius: cornerRadius))
+    }
+}
+
+struct ElevationModifier: ViewModifier {
+
+    let level: Elevation
+    let cornerRadius: CGFloat
```

Comment:
As the caller need to provide the selection border or other decorations itself anyway, the clipping should probably be not a part of `Elevation` at the end.